.github/CODEOWNERS

```diff
@@ -1,13 +1,13 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
+# Documentation with syntax examples:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
 # These owners will be the default owners for everything in the repo.
-*       @DataBiosphere/jadeteam
+*       @DataBiosphere/data-custodian-journeys
 
 # Order is important. The last matching pattern has the most precedence.
-# So if a pull request only touches javascript files, only these owners
-# will be requested to review.
-#*.js    @octocat @github/js
-
-# You can also use email addresses if you prefer.
-#docs/*  docs@example.com
+# So if a pull request only touches files in a `/snapshotbuilder` directory
+# such as `/src/main/../snapshotbuilder` or `/src/test/../snapshotbuilder`,
+# only these owners will be requested to review.
+**/snapshotbuilder @DataBiosphere/data-explorer-eng
```

Comment:
I'm open to expanding this somewhat, but I don't think we'll get it to be bulletproof: I know that cohort builder work can touch files outside of these directories.